packages/flutter_localizations/lib/src/l10n/material_kn.arb

```diff
@@ -1,55 +1,55 @@
 {
-  "scriptCategory": "tall",
-  "timeOfDayFormat": "H:mm",
-  "openAppDrawerTooltip": "ನ್ಯಾವಿಗೇಶನ್‌ ಮೆನು ತೆರೆಯಿರಿ",
-  "backButtonTooltip": "ಹಿಂತಿರುಗಿ",
-  "closeButtonTooltip": "ಮುಚ್ಚಿರಿ",
-  "deleteButtonTooltip": "ಅಳಿಸಿ",
-  "nextMonthTooltip": "ಮುಂದಿನ ತಿಂಗಳು",
-  "previousMonthTooltip": "ಹಿಂದಿನ ತಿಂಗಳು",
-  "nextPageTooltip": "ಮುಂದಿನ ಪುಟ",
-  "previousPageTooltip": "ಹಿಂದಿನ ಪುಟ",
-  "showMenuTooltip": "ಮೆನು ತೋರಿಸಿ",
-  "aboutListTileTitle": "$applicationName ಬಗ್ಗೆ",
-  "licensesPageTitle": "ಪರವಾನಗಿಗಳು",
-  "pageRowsInfoTitle": "$rowCount ರಲ್ಲಿ $firstRow–$lastRow",
-  "pageRowsInfoTitleApproximate": "$rowCount ರಲ್ಲಿ $firstRow–$lastRow",
-  "rowsPerPageTitle": "ಪ್ರತಿ ಪುಟಕ್ಕೆ ಸಾಲುಗಳು:",
-  "tabLabel": "$tabCount ರಲ್ಲಿನ $tabIndex ಟ್ಯಾಬ್",
-  "selectedRowCountTitleOne": "1 ಐಟಂ ಆಯ್ಕೆ ಮಾಡಲಾಗಿದೆ",
-  "selectedRowCountTitleOther": "$selectedRowCount ಐಟಂಗಳನ್ನು ಆಯ್ಕೆ ಮಾಡಲಾಗಿದೆ",
-  "cancelButtonLabel": "ರದ್ದುಮಾಡಿ",
-  "closeButtonLabel": "ಮುಚ್ಚಿರಿ",
-  "continueButtonLabel": "ಮುಂದುವರಿಸಿ",
-  "copyButtonLabel": "ನಕಲಿಸಿ",
-  "cutButtonLabel": "ಕತ್ತರಿಸಿ",
-  "okButtonLabel": "ಸರಿ",
-  "pasteButtonLabel": "ಅಂಟಿಸಿ",
-  "selectAllButtonLabel": "ಎಲ್ಲವನ್ನೂ ಆಯ್ಕೆಮಾಡಿ",
-  "viewLicensesButtonLabel": "ಪರವಾನಗಿಗಳನ್ನು ವೀಕ್ಷಿಸಿ",
-  "anteMeridiemAbbreviation": "ಬೆಳಿಗ್ಗೆ",
-  "postMeridiemAbbreviation": "ಸಂಜೆ",
-  "timePickerHourModeAnnouncement": "ಗಂಟೆಗಳನ್ನು ಆಯ್ಕೆಮಾಡಿ",
-  "timePickerMinuteModeAnnouncement": "ನಿಮಿಷಗಳನ್ನು ಆಯ್ಕೆಮಾಡಿ",
-  "modalBarrierDismissLabel": "ವಜಾಗೊಳಿಸಿ",
-  "signedInLabel": "ಸೈನ್ ಇನ್ ಮಾಡಲಾಗಿದೆ",
-  "hideAccountsLabel": "ಖಾತೆಗಳನ್ನು ಮರೆಮಾಡಿ",
-  "showAccountsLabel": "ಖಾತೆಗಳನ್ನು ತೋರಿಸಿ",
-  "drawerLabel": "ನ್ಯಾವಿಗೇಶನ್‌ ಮೆನು",
-  "popupMenuLabel": "ಪಾಪ್ಅಪ್ ಮೆನು",
-  "dialogLabel": "ಡೈಲಾಗ್",
-  "alertDialogLabel": "ಎಚ್ಚರಿಕೆ",
-  "searchFieldLabel": "ಹುಡುಕಿ",
-  "reorderItemToStart": "ಪ್ರಾರಂಭಕ್ಕೆ ಸರಿಸಿ",
-  "reorderItemToEnd": "ಕೊನೆಗೆ ಸರಿಸಿ",
-  "reorderItemUp": "ಮೇಲೆ ಸರಿಸಿ",
-  "reorderItemDown": "ಕೆಳಗೆ ಸರಿಸಿ",
-  "reorderItemLeft": "ಎಡಕ್ಕೆ ಸರಿಸಿ",
-  "reorderItemRight": "ಬಲಕ್ಕೆ ಸರಿಸಿ",
-  "expandedIconTapHint": "ಕುಗ್ಗಿಸಿ",
-  "collapsedIconTapHint": "ವಿಸ್ತರಿಸಿ",
-  "remainingTextFieldCharacterCountOne": "1 ಅಕ್ಷರ ಉಳಿದಿದೆ",
-  "remainingTextFieldCharacterCountOther": "$remainingCount ಅಕ್ಷರಗಳು ಉಳಿದಿವೆ",
-  "refreshIndicatorSemanticLabel": "ರಿಫ್ರೆಶ್ ಮಾಡಿ",
-  "moreButtonTooltip": "TBD"
+  "scriptCategory": "\u0074\u0061\u006c\u006c",
```

Comment:
I'm surprised this was not encoded before, since I thought we did to avoid crashing some emacs users. Does this seem right @HansMuller?

Reply:
Seems like this was undone at https://github.com/flutter/flutter/commit/4841a7ed6fb36774974d6cf2aafb7f791908ab8d#diff-8fc39fc19a4bf5d7bcd765ed163fd9ee.

Filed an issue (https://github.com/flutter/flutter/issues/53449) so that we should automatically incorporate this utility into the existing gen_date_localizations.dart and gen_localizations.dart scripts

Reply:
Thanks for tracking that down, yes this file should always be encoded.